host/src/server/api/v3/proof/aggregate/prune.rs

```diff
@@ -0,0 +1,33 @@
+use axum::{debug_handler, extract::State, routing::post, Router};
+use raiko_tasks::TaskManager;
+use utoipa::OpenApi;
+
+use crate::{interfaces::HostResult, server::api::v2::PruneStatus, ProverState};
+
+#[utoipa::path(post, path = "/proof/aggregate/prune",
+    tag = "Proving",
+    responses (
+        (status = 200, description = "Successfully pruned all aggregation tasks", body = PruneStatus)
+    )
+)]
+#[debug_handler(state = ProverState)]
+/// Prune all aggregation tasks.
+async fn prune_handler(State(prover_state): State<ProverState>) -> HostResult<PruneStatus> {
+    let mut manager = prover_state.task_manager();
+
+    manager.prune_aggregation_db().await?;
```

Comment:
does the prune clear all status?? so next query req generates a new proof (same) again?

Reply:
Yes, it clears all statuses. Next call is like no requests have been sent before